### PR TITLE
Remove shadow scroll

### PR DESCRIPTION
**Pull request by hanbyul-here**
Handles issue in https://github.com/NASA-IMPACT/veda-ui/issues/682#issuecomment-1747334303

While putting this fix, I noticed that `use-media-query` returns document body size, which differs from how CSS media query works. This results in a weird style in between 991 px ~ 1000 px. (screenshot attached, it is on ghg.center now)  👉  I didn't fix this problem, but I patched it with CSS.

(Eventually, we would need to move to Container query instead of media query: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Container_Queries) 

Screen Shot 2023-10-04 at 3 01 10 PM

### CODE REVIEW

File: app/scripts/components/common/shadow-scrollbar-improved.tsx [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import React, { useCallback, useEffect, useState } from 'react';
 import { ShadowScrollbar } from '@devseed-ui/shadow-scrollbar';
 
+// Moving away from this component because of the issue
+// https://github.com/NASA-IMPACT/veda-ui/issues/682#issuecomment-1747334303
```
Comment (nerik) [inline app/scripts/components/common/shadow-scrollbar-improved.tsx:5]: I think you can go ahead and remove the file. Juste use this comment as the commit message.

File: app/scripts/components/common/mapbox/layer-legend.tsx [hunk 2]
```diff
@@ -252,17 +251,11 @@ export function LayerLegend(
...
       renderBody={() => (
         <LegendBody>
-          <ShadowScrollbar
-            scrollbarsProps={{
-              autoHeight: true,
-              autoHeightMin: 32,
-              autoHeightMax: 120
-            }}
-          >
+
```
Comment (nerik) [inline app/scripts/components/common/mapbox/layer-legend.tsx:254]: Text is cut off here:

Screenshot 2023-10-05 at 14 38 07

On `data-catalog/sedac-popdensity-yeargrid5yr-v4.11`

**Review by nerik — CHANGES_REQUESTED** [review]
Looking good, thanks 👍 
Please review the issue with the map legend.

**Review by nerik — APPROVED** [review]

**Comment by hanbyul-here** [comment]
Ah looks like I missed the case when there are many layers. working on the fix now. (for PanelBody)

Screen Shot 2023-10-05 at 2 11 46 PM

**Pull request merged**